lib/wings/model_transformer.rb

```diff
@@ -54,7 +54,8 @@ def self.relationship_keys_for(reflections:)
                       .keys
                       .reject { |k| k.to_s.include?('id') }
                       .map { |k| k.to_s.singularize + '_ids' }
-      relationships.delete(:member_ids) # Remove here.  Members will be extracted as ordered_members in attributes method.
+      relationships.delete('member_ids') # Remove here.  Members will be extracted as ordered_members in attributes method.
```

Comment:
This wasn't actually being deleted since the keys are strings, not symbols.

Reply:
huh. do we know if it needed to be? i suppose your new tests fail without this line?